app/views/content_items/service_manual_topic.html.erb

```diff
@@ -5,11 +5,12 @@
   <%= render 'shared/custom_phase_message', phase: @content_item.phase %>
 <% end %>
 
-  <%= render "govuk_publishing_components/components/breadcrumbs", {
-    breadcrumbs: @content_item.breadcrumbs,
-    collapse_on_mobile: true
-  } %>
+<% content_for :header do %>
+  <%= render partial: "content_items/service_manual_breadcrumbs" %>
+<% end %>
 
+<div class="govuk-width-container">
```

Comment:
I compared this file to the two similar files: `service_manual_service_standard.html.erb` and `service_manual_guide.html.erb` and it seems this page lacks the govuk-width-container div which is required by the design system. I've added it. This causes a change in the left and right margins, but it makes it consistent with other pages.